core/crypto/vrf/vrf_provider_impl.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/vrf/vrf_provider_impl.hpp"
+
+namespace kagome::crypto {
+
+  VRFKeypair VRFProviderImpl::generateKeypair() const {
+    size_t initseed = std::random_device()();
+    std::mt19937 gen(initseed);
+    std::vector<uint8_t> seed(SR25519_SEED_SIZE, 0);
+    std::generate(seed.begin(), seed.end(),
+                  [&gen]() { return static_cast<uint8_t>(gen()); });
+
+    std::vector<uint8_t> kp(SR25519_KEYPAIR_SIZE, 0);
+    sr25519_keypair_from_seed(kp.data(), seed.data());
+
+    VRFKeypair result{};
+    std::copy(kp.begin(), kp.begin() + SR25519_SECRET_SIZE,
+              result.secret_key.begin());
+    std::copy(kp.begin() + SR25519_SECRET_SIZE,
+              kp.begin() + SR25519_SECRET_SIZE + SR25519_PUBLIC_SIZE,
+              result.public_key.begin());
+
+    return result;
+  }
+
+  std::array<uint8_t, 32> uint256_t_to_bytes(
+      boost::multiprecision::uint256_t const &i) {
+    auto count = i.backend().size();
+    auto tsize = sizeof(boost::multiprecision::limb_type);
+    auto copy_count = count * tsize;
+
+    std::array<uint8_t, 32> output{};
+    memcpy(output.data(), i.backend().limbs(), copy_count);
+    //    if (len > copy_count) {
+    //      memset(output + copy_count, 0, len - copy_count);
+    //    }
```

Comment:
Remove or uncomment, please

Reply:
Probably we need to implement more random numbers generators in order to use common way for generating sequences.